src/app/state/courses/courses.actions.ts

```diff
@@ -0,0 +1,39 @@
+import { createAction, props } from '@ngrx/store';
+import { Course } from 'src/app/models/data-model';
+
+export const getCoursesSuccess = createAction(
+    'Get Courses Success',
+    props<{courses: Course[]}>(), 
+);
+
+export const getSomeCourses = createAction(
+    'Get Some Courses',
+    props<{courses: Course[]}>(),
+);
+
+export const getCourses = createAction(
+    'Get Courses',
+);
+
+export const getAllCourses = createAction(
```

Comment:
The action seems to be redundant. Please remove it to meet the clean code standards